.gitlab-ci.yml

```diff
@@ -1,12 +1,35 @@
 stages:
+  - vulnerabilities
   - test
 
-before_script:
-  - curl -sS https://getcomposer.org/installer | php
-  - php composer.phar install --no-scripts --quiet --ignore-platform-reqs
-  - cp app/config/parameters.yml.dist app/config/parameters.yml
```

Comment:
is dit niet meer nodig dan?

Reply:
Nee, als die hier staat wordt die bij elke job uitgevoerd. Maar door de andere image is dat niet meer nodig. En bij die voor de php unit tests stond er toch een specifieke gedefinieerd.